test/e2e/node/crictl.go

```diff
@@ -18,11 +18,11 @@ package node
 
 import (
 	"fmt"
-	"strings"
 
 	"k8s.io/kubernetes/test/e2e/framework"
+	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
-	e2essh "k8s.io/kubernetes/test/e2e/framework/ssh"
+	"k8s.io/kubernetes/test/e2e/storage/utils"
```

Comment:
side note, @oomichi we should start thinking in moving the `NewHostExec` to a more general location, right?
I start to see it in more places than in storage:
```
test/e2e/node/mount_propagation.go:             hostExec := utils.NewHostExec(f)
test/e2e/storage/utils/host_exec.go:// NewHostExec returns a HostExec
test/e2e/storage/utils/host_exec.go:func NewHostExec(framework *framework.Framework) HostExec {
test/e2e/storage/persistent_volumes-local.go:           hostExec := utils.NewHostExec(f)
test/e2e/storage/testsuites/volumemode.go:              hostExec := storageutils.NewHostExec(f)
test/e2e/storage/testsuites/subpath.go:         l.hostExec = utils.NewHostExec(f)
test/e2e/storage/drivers/in_tree.go:    l.hostExec = utils.NewHostExec(f)
test/e2e/network/netpol/network_legacy.go:      hostExec := utils.NewHostExec(f)
test/e2e/network/util.go:       hostExec := utils.NewHostExec(f)
test/e2e/network/service.go:            hostExec := utils.NewHostExec(f)
test/e2e/network/service.go:            hostExec := utils.NewHostExec(f)
```
cc: @BenTheElder

Reply:
Yeah I thought the same. I think I could follow-up on that and move it into a more dedicated location.

Reply:
yeah, follow up, not drive by, just saw it and added a comment so I don't forget